docs/pages/x/api/date-pickers/desktop-date-range-picker.json

```diff
@@ -5,10 +5,7 @@
       "type": { "name": "enum", "description": "1<br>&#124;&nbsp;2<br>&#124;&nbsp;3" },
       "default": "2"
     },
-    "closeOnSelect": {
-      "type": { "name": "bool" },
-      "default": "`true` for desktop, `false` for mobile (based on the chosen wrapper and `desktopModeMediaQuery` prop)."
-    },
+    "closeOnSelect": { "type": { "name": "bool" }, "default": "`true`" },
```

Comment:
The changes on the prop description are not exactly related to the task, but I used it to define more specific description and avoid confusions. In this case, for example, we were mentioning differences on default value for desktop and mobile on the API ref that is dedicated to a desktop component

Reply:
Great use of the opportunity. 👍 
It's more needed and useful after this behavioral change. 👌